tailwind.config.ts

```diff
@@ -12,6 +12,7 @@ export default {
       colors: {
         background: "var(--background)",
         foreground: "var(--foreground)",
+        primary: "#ba43f7"
```

Comment:
Let's use the `mauve` color in the [catppuccin](https://github.com/catppuccin/catppuccin) palette